File: sms-api-java/send-message-service/src/main/java/com/ffos/sendmessageservice/SendMessageServiceApplication.java
package com.ffos.sendmessageservice;

import org.modelmapper.ModelMapper;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
import org.springframework.context.annotation.Bean;
import org.springframework.http.HttpMethod;
import org.springframework.web.reactive.function.client.WebClient;

@SpringBootApplication
@EnableDiscoveryClient
public class SendMessageServiceApplication {

	public static void main(String[] args) {
		SpringApplication.run(SendMessageServiceApplication.class, args);
	}

	@Bean
	public ModelMapper modelMapper(){
		return new ModelMapper();
	}

}

File: sms-api-kotlin/sendmessageservice/src/main/kotlin/com/example/sendmessageservice/SendmessageserviceApplication.kt
package com.example.sendmessageservice

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@SpringBootApplication
@EnableDiscoveryClient
class SendmessageserviceApplication

fun main(args: Array<String>) {
	runApplication<SendmessageserviceApplication>(*args)
}
